test/framework/workload.go

```diff
@@ -8,13 +8,13 @@ type WorkloadCluster struct {
 type WorkloadClusters map[string]*WorkloadCluster
 
 func (w *WorkloadCluster) CreateCluster() {
-	w.createCluster(withKubeconfig(w.managementClusterKubeconfigFile()))
+	w.createCluster(nil, withKubeconfig(w.managementClusterKubeconfigFile()))
 }
 
 func (w *WorkloadCluster) UpgradeCluster(opts ...ClusterE2ETestOpt) {
-	w.upgradeCluster([]commandOpt{withKubeconfig(w.managementClusterKubeconfigFile())}, opts...)
+	w.upgradeCluster(opts, []CommandOpt{withKubeconfig(w.managementClusterKubeconfigFile())}...)
```

Comment:
you could just do `w.upgradeCluster(opts, withKubeconfig(w.managementClusterKubeconfigFile()))` (I think)